internal/command/jsonformat/change/change.go

```diff
@@ -105,3 +106,12 @@ func (change Change) unchanged(keyword string, count int) string {
 	}
 	return fmt.Sprintf("[dark_gray]# (%d unchanged %ss hidden)[reset]", count, keyword)
 }
+
+// escapeAttributeName checks if `name` contains any HCL syntax and returns it
+// surrounded by quotation marks if it does.
+func (change Change) escapeAttributeName(name string) string {
```

Comment:
This name misled me briefly, as it doesn't _always_ escape the attribute name. Something like `maybeEscapeAttributeName` or `ensureValidAttributeName` or similar might get across the intention a bit better.

Reply:
Done!